frontend/src/pages/calculatedResults/index.js

```diff
@@ -1,13 +1,62 @@
 import React from 'react';
-import { PlaceholderStyle } from '../../components/placeholderStyle/PlaceholderStyleElement';
-import './CalculatedResultsElements';
+import { useLocation } from 'react-router-dom';
+import Footer from '../../components/footerPurple';
+import {
+  FixedInstallmentsContainer,
+  InstallmentContainer,
+  IllustrationContainer,
+  DescendingInstallmentsContainer,
+  YearTabs,
+} from './CalculatedResultsElements';
+import { SectionTitle } from '../../components/sectionTitle/SectionTitle';
+import { GradientText } from '../../components/gradientText/GradientText';
+import landingPageIllustration from '../../images/landingPageIllustration.svg';
+
+const { TabPane } = YearTabs;
 
 function CalculatedResults() {
+  const location = useLocation();
+
+  if (location.state.response.monthly_payment.length !== undefined) {
+    return (
+      <>
+        <DescendingInstallmentsContainer>
+          <YearTabs defaultActiveKey="1" centered>
+            <TabPane tab="Rok 1" key="1">
```

Comment:
(optional) Sometimes when code blocks are repeated, it is nice to add simple `forEach` or `map`

Reply:
definitely will do when real data is available